### PR TITLE
 support for super user

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

temporary support for super user 

1.  only  auth  by JWT 
2.  temporary bypass current Privilege system (fixed Privileges, no need to store/get in meta)

@flaneur2020 @ZhiHanZ 

by adding is_superuser field,  we have a sep user namespace for  superusers.
this is the way with minimal code change. 

btw, we already has an UserPrivilegeType called `Super`

```
    // Privilege to Kill query, Set global configs, etc.
    Super = 1 << 9,
```

- [ ] add unit test

## Changelog

- Not for changelog (changelog entry is not required)

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: query/src/users/auth/auth_mgr.rs [hunk 1]
```diff
@@ -56,11 +56,15 @@ impl AuthMgr {
...
         match credential {
             Credential::Jwt { token: t } => {
-                let user_name = match &self.jwt {
+                let (user_name, is_supper) = match &self.jwt {
                     Some(j) => j.get_user(t.as_str())?,
                     None => return Err(ErrorCode::AuthenticateFailure("jwt auth not configured.")),
                 };
-                self.users.get_user(&self.tenant, &user_name, "%").await
+                if is_supper {
+                    Ok(UserInfo::new_supper(user_name))
```
Comment (flaneur2020) [inline query/src/users/auth/auth_mgr.rs:64]: maybe the super user's granted privileges would be empty in this case 🤔

some unit tests may help us making the semantics of superuser more clear

Comment (youngsofun) [reply query/src/users/auth/auth_mgr.rs:64]: > maybe the super user's granted privileges would be empty in this case 🤔
it was default(), I just changed to empty()

Comment (flaneur2020) [reply query/src/users/auth/auth_mgr.rs:64]: maybe the superuser has more privileges than root?  q.q

if it's root user, it'd get all the privileges

Comment (youngsofun) [reply query/src/users/auth/auth_mgr.rs]: I though superuser do not use privileges at all.

**Comment by flaneur2020** [comment]
how about limit the scope of the super marker, instead add an user option like `can modify tenant`?

**Comment by ZhiHanZ** [comment]
Additionally, this PR is related to daily ops, and need some example documentations about JWT and how to access databend-query as super user with JWT Authentication

**Pull request closed without merging**